tests/integration/replay/replay_tests.cpp

```diff
@@ -27,6 +28,17 @@ class replayTest {
   {
     auto& rm = umpire::ResourceManager::getInstance();
 
+    allocatorNames.push_back("HOST");
+#if defined(UMPIRE_ENABLE_DEVICE)
+    allocatorNames.push_back("DEVICE");
+#endif
+#if defined(UMPIRE_ENABLE_UM)
+    allocatorNames.push_back("UM");
+#endif
+#if defined(UMPIRE_ENABLE_PINNED)
+    allocatorNames.push_back("PINNED");
+#endif
+
```

Comment:
It is easy enough to generate different comparison files for different configurations.

The tests need to somehow use a different "good" file to compare to based upon the compile-time options above.  The comparison file is currently specified by a bash script.  Are there flags that we can pass in to the bash script that will allow the script to dynamically use a different comparison file based upon what devices and capabilities are present?

Reply:
Yup, you should be able to pass arguments based on the CMake variables to the bash script. In CMake, the variables won't have the `UMPIRE_` prefix.

Reply:
This will be addressed via a separate PR [UM-419](https://rzlc.llnl.gov/jira/browse/UM-419)